.gitignore

```diff
@@ -16,4 +16,7 @@
 .tox/
 .pytest_cache/
 .coverage
-htmlcov/
\ No newline at end of file
+_build/
```

Comment:
Given that `_build`, `_static` and `_templates` are all of them related with documentation, maybe it can be grouped together within the same folder (e.g. `_doc`) to avoid polluting too much the root folder.

Reply:
I guess apart from `_build`, the other directories are created automatically by sphinx.

Reply:
Changing the paths here
https://github.com/scrapinghub/scrapy-poet/pull/21/files#diff-85987f48f1258d9ee486e3191495582dR50 
and here:
https://github.com/scrapinghub/scrapy-poet/pull/21/files#diff-85987f48f1258d9ee486e3191495582dR93
wouldn't work?

Reply:
It would work, but I'm not sure it's a regular practice. Scrapy, for example, keeps the original value for this config: https://github.com/scrapy/scrapy/blob/master/docs/conf.py

Reply:
Although it commits `_static` and `_templates` directories 🤔

Reply:
I think I missed that `_build`, `_static` and `_templates` are going to be created inside the `docs` folder, right?  If this is the case then my proposal makes no sense. 

Besides, I don't know if these folders should committed. I see that scrappy does not include `_static` and `_templates`  in `.gitignore` but it does for `docs/build` instead (the equivalent to `_build` in your PR).

Reply:
By the way, I see this is already merged! Forget about that :+1: